src/odemis/dataio/tiff.py

```diff
@@ -2722,3 +2744,168 @@ def _convertTFStoOdemisMD(metadata: str) -> dict:
         logging.warning(f"Failed to parse ThermoFisher metadata: {model.MD_DET_TYPE} - {e}")
 
     return md
+
+## TESCAN
+
+def _parse_tescan_metadata(metadata: str) -> dict:
+    """
+    Parses TESCAN metadata (ini) to dictionary
+    :param metadata: metadata string as ini (TESCAN format)
+    :return: (dict) metadata as dictionary (TESCAN format)
+    """
+    # NOTE:
+    # The metadata should be formatted as ini, but seems to be corrupted
```

Comment:
yeah, you are correct good call. i've changed the parser to account for this (skip jpeg2000 metadata, then read chunks + headers  sequentially). It now can be read directly with configparser so that also removes the manual parsing to dict.